nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,13 +9,26 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 1.1.0
+***************
+
+* The PDN socket has been deprecated.
+* Added the possibility to specify the PDN ID to bind a socket to with the ``NRF_SO_BINDTODEVICE`` socket option.
+* Added the ``NRF_AI_PDNSERV`` flag for :c:func:`nrf_getaddrinfo` to specify the ID of the PDN to route a DNS query through.
+* Added the ``NRF_SO_SEC_DTLS_HANDSHAKE_TIMEO`` socket option to set the DTLS handshake timeout.
+* Added the ``NRF_SO_SEC_SESSION_CACHE_PURGE`` socket option to purge TLS/DTLS session cache.
+* Updated :c:func:`nrf_connect` to set ``errno`` to ``NRF_ECONNREFUSED`` when failing due to a missing or wrong certificate.
```

Comment:
Is `NRF_ECONNREFUSED` also given when using wrong PSK?

Reply:
No, but it also applies when failing to parse the Private Key. @umapraseeda could we add that?

Reply:
Sure..I shall add that.